### PR TITLE
[Liquibase]: documentation for the Ydb Dialect of Liquibase

### LINKED ISSUE #73: dev:  liquibase dialect documentation en / ru in ydb.tech

### PR DESCRIPTION
**Pull request by KirillKurdyukov**
### Changelog category 

* Documentation (changelog entry is not required)

### CODE REVIEW

File: ydb/docs/ru/core/integrations/liquibase.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
+## Поддержка диалекта YDB в инструменте миграции Liquibase. ##
```
Comment (Pseudolukian) [inline ydb/docs/ru/core/integrations/liquibase.md:1]: Точки в заголовках не ставятся. Тут такая же история с уровнями заголовков и якорями как и в Hibernate.

File: ydb/docs/ru/core/integrations/liquibase.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+
+### Введение ###
```
Comment (Pseudolukian) [inline ydb/docs/ru/core/integrations/liquibase.md:3]: Давай сократим и оставим только суть:

Liquibase – это библиотека с открытым исходным кодом для отслеживания, управления и применения изменений схемы базы данных. Liquibase расширяется диалектами. 

Диалект – это ... Мы разработали YDB диалект. Далее переходим к разделу Возможности диалекта YDB.

File: ydb/docs/ru/core/integrations/liquibase.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+
+В современном мире разработки ПО управление версиями схемы базы данных стало критически важной задачей для обеспечения согласованности и отката изменений. Инструменты миграции базы данных, такие как Liquibase предоставляют средства для версионирования, отслеживания и применения изменений схемы.
+
+Однако, появление новых СУБД ставит задачу интеграции с этими системами для поддержки их уникальных возможностей и диалектов SQL.
+
+Как раз о результатах интеграции диалекта YDB c Liquibase мы поговорим в этой статье.
+
+### Возможности диалекта YDB ###
```
Comment (Pseudolukian) [inline ydb/docs/ru/core/integrations/liquibase.md:11]: Нужны примеры создания/удаления/изменения таблиц.

File: ydb/docs/ru/core/integrations/liquibase.md [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+
+В диалекте поддержаны такие основные конструкции:
+
+1. `createTable` создание таблицы. Описание типов из SQL стандарта сопоставляется с примитивными типами YDB. К примеру тип bigint будет конвертирован в Int64. Также можно явно указывать оригинальное название, например, такие типы как Int32, Json, JsonDocument, Bytes, Interval. Но в таком случае теряется переносимость схемы.
+2. `dropTable`, `alterTable`, `createIndex`, `addColumn`. Соответственно удаление таблицы, изменение, создание индексов и добавление колонки.
+3. `loadData`, `loadUpdateData`. Загрузка данных из CSV файлов.
+
+Чтобы понять, какие SQL-конструкции может выполнять YDB, ознакомьтесь с документацией по языку запросов [YQL](https://ydb.tech/docs/ru/yql/reference/).
+
+Важно отметить, что кастомные инструукции YQL можно накатывать через нативные SQL запросы. Но YDB не стоит на месте! В дальнейшем все больше и больше классических SQL конструкций будут поддержаны.
```
Comment (Pseudolukian) [inline ydb/docs/ru/core/integrations/liquibase.md:23]: Опять же нужен пример. Слово "накатывать" нужно заменить. Это, конечно, круто, что YDB не стоит на месте, но лучше писать только о том, что уже реализовано. Нужно удалить:
"Но YDB не стоит на месте! В дальнейшем все больше и больше классических SQL конструкций будут поддержаны".

File: ydb/docs/ru/core/integrations/liquibase.md [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+
+### Как воспользоваться? ###
```
Comment (Pseudolukian) [inline ydb/docs/ru/core/integrations/liquibase.md:25]: Я бы объединил Как воспользоваться и Пример использования диалекта.

**Review by Pseudolukian — DISMISSED** [review]

File: ydb/docs/ru/core/integrations/liquibase.md (hunk 3 above)
Comment (KirillKurdyukov) [reply ydb/docs/ru/core/integrations/liquibase.md:11]: А примеры ж ниже?

File: ydb/docs/ru/core/integrations/liquibase.md (hunk 5 above)
Comment (KirillKurdyukov) [reply ydb/docs/ru/core/integrations/liquibase.md]: Объедил но все - таки h3 оставил, чтобы выделить начало примера

File: ydb/docs/en/core/integrations/liquibase.md [hunk 6]
```diff
@@ -0,0 +1,273 @@
+# Support for the YDB dialect in the Liquibase migration tool.
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:1]:
```suggestion
# Migrating {{ ydb-short-name }} data schemas with the Liquibase migration tool
```

File: ydb/docs/en/core/integrations/liquibase.md [hunk 7]
```diff
@@ -0,0 +1,273 @@
+# Support for the YDB dialect in the Liquibase migration tool. 
+
+## Introduction 
+
+In today's world of software development, database schema versioning is a critical task to ensure consistency and the ability to roll back changes. Database migration tools, such as Liquibase, provide tools for versioning, tracking, and applying changes to the schema.
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:5]: Given the term "consistency" is used so extensively in the context of consistency models and anomalies (like in https://jepsen.io/consistency), I'd recommend to rephrase this paragraph to avoid using it in the other meaning.

Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:5]:
```suggestion
In today's world of software development, database schema versioning is a critical task to ensure consistency and the ability to roll back changes. Database migration tools, such as [Liquibase](https://www.liquibase.com/), provide tools for versioning, tracking, and applying changes to the schema. 
```

Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:5]: Also, an introduction in documentation needs to be more practical. Like what you could achieve if you use this specific integration. Readers here already have some understanding about what is YDB and might have already stumbled upon migration issues.

File: ydb/docs/en/core/integrations/liquibase.md [hunk 8]
```diff
@@ -0,0 +1,273 @@
+# Support for the YDB dialect in the Liquibase migration tool. 
+
+## Introduction 
+
+In today's world of software development, database schema versioning is a critical task to ensure consistency and the ability to roll back changes. Database migration tools, such as Liquibase, provide tools for versioning, tracking, and applying changes to the schema. 
+
+However, the emergence of new database management systems (DBMS) has made it necessary to integrate with these systems in order to support their unique capabilities and SQL dialects. 
+
+We will discuss the results of integrating DB dialect with Liquibase in this article.
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:9]:
```suggestion
```
this might be ok as an introduction to a blog post, but looks irrelevant for documentation

File: ydb/docs/en/core/integrations/liquibase.md [hunk 9]
```diff
@@ -0,0 +1,273 @@
...
+
+## Features of the YDB Dialect
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:11]:
```suggestion
## Features of the {{ ydb-short-name }} Dialect 
```

File: ydb/docs/en/core/integrations/liquibase.md [hunk 10]
```diff
@@ -0,0 +1,273 @@
...
+
+In today's world of software development, database schema versioning is a critical task to ensure consistency and the ability to roll back changes. Database migration tools, such as Liquibase, provide tools for versioning, tracking, and applying changes to the schema. 
+
+However, the emergence of new database management systems (DBMS) has made it necessary to integrate with these systems in order to support their unique capabilities and SQL dialects. 
+
+We will discuss the results of integrating DB dialect with Liquibase in this article.
+
+## Features of the YDB Dialect 
+
+Liquibase's main functionality is the abstract description of database schemas in `.xml`, `.json`, and `.yaml` formats. This ensures portability when switching between different database management systems (DBMS).
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:13]:
```suggestion
Liquibase's main functionality is the abstract description of database schemas in `.xml`, `.json`, or `.yaml` format. This ensures portability when switching between different database management systems (DBMS).
```

File: ydb/docs/en/core/integrations/liquibase.md [hunk 11]
```diff
@@ -0,0 +1,273 @@
...
+
+The following basic features are supported in the YDB dialect:
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:15]:
```suggestion
The following basic features are supported in the {{ ydb-short-name }} dialect:
```

File: ydb/docs/en/core/integrations/liquibase.md (hunk 7 above)
Comment (blinkov) [reply ydb/docs/en/core/integrations/liquibase.md:5]: the same applies for the rest of the text

**Review by blinkov — CHANGES_REQUESTED** [review]

File: ydb/docs/en/core/integrations/liquibase.md [hunk 12]
```diff
@@ -0,0 +1,314 @@
...
+    </changeSet>
+    <changeSet id="index_episodes_title" author="kurdyukov-kir">
+        <createIndex tableName="episodes" indexName="index_episodes_title" unique="false">
+            <column name="title"/>
+        </createIndex>
+    </changeSet>
+</databaseChangeLog>
+```
+
+After executing the `liquibase update` command, Liquibase will print the following log.
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:172]:
```suggestion
After executing the `liquibase update` command, Liquibase will print the following log:
```

File: ydb/docs/en/core/integrations/liquibase.md [hunk 13]
```diff
@@ -0,0 +1,314 @@
...
+-------------------------------
+Total change sets:            2
+
+Liquibase: Update has been successful. Rows affected: 2
+Liquibase command 'update' was executed successfully.
+```
+
+After applying migrations, the data schema now looks like this:
+
+![../_assets/liquibase-step-1.png](../_assets/liquibase-step-1.png)
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:188]: An `_assets` folder can be on any level in the documentation, and it's better to move these screenshots closer to the article to the `integrations` folder.

File: ydb/docs/en/core/integrations/liquibase.md [hunk 14]
```diff
@@ -0,0 +1,314 @@
...
+
+You can see that Liquibase has created two service tables: `DATABASECHANGELOG`, which is the migration log, and `DATABASECHANGELOGLOCK`, which is a table for acquiring a distributed lock.
+
+Contents of the `DATABASECHANGELOG` table:
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:192]:
```suggestion
You can see that Liquibase has created two service tables: `DATABASECHANGELOG`, which is the migration log, and `DATABASECHANGELOGLOCK`, which is a table for acquiring a distributed lock.

Example contents of the `DATABASECHANGELOG` table:
```

File: ydb/docs/en/core/integrations/liquibase.md [hunk 15]
```diff
@@ -0,0 +1,314 @@
...
+### Evolution of the database schema
+
+Let's say we need to create a {{ydb-short-name }} topic and turn off auto partitioning of the table. This can be done with a native SQL script:
+
+```sql
+--liquibase formatted sql
+
+--changeset kurdyukov-kir:10
+CREATE
+TOPIC `my_topic` (
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:208]:
```suggestion
CREATE TOPIC `my_topic` (
```

File: ydb/docs/en/core/integrations/liquibase.md [hunk 16]
```diff
@@ -0,0 +1,314 @@
...
+
+| AUTHOR        | COMMENTS        | CONTEXTS | DATEEXECUTED | DEPLOYMENT_ID | DESCRIPTION                                                    | EXECTYPE | FILENAME               | ID                   | LABELS | LIQUIBASE | MD5SUM                             | ORDEREXECUTED | TAG |
+|:--------------|:----------------|:---------|:-------------|:--------------|:---------------------------------------------------------------|:---------|:-----------------------|:---------------------|:-------|:----------|:-----------------------------------|:--------------|:----|
+| kurdyukov-kir | Table episodes. |          | 12:53:27     | 1544007500    | createTable tableName=episodes                                 | EXECUTED | migration/episodes.xml | episodes             |        | 4.25.1    | 9:4067056a5ab61db09b379a93625870ca | 1             |
+| kurdyukov-kir | ""              |          | 12:53:28     | 1544007500    | createIndex indexName=index_episodes_title, tableName=episodes | EXECUTED | migration/episodes.xml | index_episodes_title |        | 4.25.1    | 9:49b8b0b22d18c7fd90a3d6b2c561455d | 2             |
+
+### Evolution of the database schema
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:199]:
```suggestion
### Database schema evolution
```

File: ydb/docs/en/core/integrations/liquibase.md [hunk 17]
```diff
@@ -0,0 +1,314 @@
...
+
+Let's say we need to create a {{ydb-short-name }} topic and turn off auto partitioning of the table. This can be done with a native SQL script:
+
+```sql
+--liquibase formatted sql
+
+--changeset kurdyukov-kir:10
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:206]: It's not clear if this number has any meaning. Also, why does it use your login for the name instead of something like `create-a-topic`?

File: ydb/docs/en/core/integrations/liquibase.md [hunk 18]
```diff
@@ -0,0 +1,314 @@
...
+TOPIC `my_topic` (
+    CONSUMER my_consumer
+    ) WITH (retention_period = Interval('P1D')
+);
+
+--changeset kurdyukov-kir:auto-partitioning-disabled
+ALTER TABLE episodes SET (AUTO_PARTITIONING_BY_SIZE = DISABLED);
+```
+
+Also add a new column `is_deleted` and remove the index `index_episodes_title`:
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:217]:
```suggestion
Also, let's add a new column `is_deleted` and remove the `index_episodes_title` index:
```

File: ydb/docs/en/core/integrations/liquibase.md [hunk 19]
```diff
@@ -0,0 +1,314 @@
...
+    <dropIndex tableName="episodes" indexName="index_episodes_title"/>
+
+    <addColumn tableName="episodes">
+        <column name="is_deleted" type="bool"/>
+    </addColumn>
+</changeSet>
+<include file="/migration/sql/yql.sql" relativeToChangelogFile="true"/>
+```
+
+After executing `liquibase update`, the database schema will be successfully updated:
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:233]:
```suggestion
After executing `liquibase update`, the database schema will be successfully updated with all of these changes:
```

File: ydb/docs/en/core/integrations/liquibase.md [hunk 20]
```diff
@@ -0,0 +1,314 @@
...
+Liquibase command 'update' was executed successfully.
+```
+
+The result will be deleting the index, adding the `is_deleted` column, disabling the auto partitioning setting, and creating a topic:
+
+![../_assets/liquibase-step-2.png](../_assets/liquibase-step-2.png)
+
+### Initializing liquibase in a project with a non-empty data schema
+
+Suppose I have an existing project with the current database schema:
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:253]:
```suggestion
Let's suppose there's an existing project with the following database schema:
```

File: ydb/docs/en/core/integrations/liquibase.md [hunk 21]
```diff
@@ -0,0 +1,314 @@
...
+
+![../_assets/liquibase-step-3.png](../_assets/liquibase-step-3.png)
+
+To start using liquibase, you need to run `liquibase generate-changelog --changelog-file=changelog.xml`.
+
+The content of the generated changelog.xml:
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:259]:
```suggestion
The contents of the generated changelog.xml:
```

File: ydb/docs/en/core/integrations/liquibase.md [hunk 22]
```diff
@@ -0,0 +1,314 @@
...
+
+![../_assets/liquibase-step-2.png](../_assets/liquibase-step-2.png)
+
+### Initializing liquibase in a project with a non-empty data schema
+
+Suppose I have an existing project with the current database schema:
+
+![../_assets/liquibase-step-3.png](../_assets/liquibase-step-3.png)
+
+To start using liquibase, you need to run `liquibase generate-changelog --changelog-file=changelog.xml`.
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:257]:
```suggestion
In this case? to start using Liquibase, you need to run `liquibase generate-changelog --changelog-file=changelog.xml` first.
```

File: ydb/docs/en/core/integrations/liquibase.md [hunk 23]
```diff
@@ -0,0 +1,314 @@
...
+    </createTable>
+</changeSet>
+<changeSet author="kurdyukov-kir (generated)" id="1711556283305-3">
+    <createIndex indexName="title_index" tableName="episodes">
+        <column name="title"/>
+    </createIndex>
+</changeSet>
+```
+
+Then you need to synchronize the generated changelog.xml the file, this is done by the command `liquibase changelog-sync --changelog-file=dbchangelog.xml`.
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:310]: It's not clear what this is supposed to do: why the filename is different? We now have two changelog files? If so, what's the difference between them?

**Review by blinkov — CHANGES_REQUESTED** [review]

File: ydb/docs/en/core/integrations/liquibase.md [hunk 24]
```diff
@@ -0,0 +1,314 @@
...
+
+Asynchronous indexes should be created using native SQL migrations.
+
+{% endnote %}
+
+`dropIndex` - drop index. For example: `<dropIndex tableName="series" indexName="series_index"/>`
+
+### Inserting data into a table
+
+`loadData`, `loadUpdateData` - download data from CSV file. *loadUpdate* loads data using the UPSERT command. Data will be automatically converted to the required types, taking into account the strict typing in {{ ydb-short-name }}.
```
[the remark this answers is not in the document]
Comment (KirillKurdyukov) [reply ydb/docs/en/core/integrations/liquibase.md:81]: loadUpdate === loadUpdateData

File: ydb/docs/en/core/integrations/liquibase.md (hunk 17 above)
Comment (KirillKurdyukov) [reply ydb/docs/en/core/integrations/liquibase.md:206]: kurdyukov-kir is author, 10 is id

File: ydb/docs/en/core/integrations/liquibase.md [hunk 25]
```diff
@@ -0,0 +1,314 @@
...
+| `timestamp`, `java.sql.Types.TIMESTAMP`, `java.sql.TIMESTAMP`                                                                                                                                                                                                                                                                                                                                                                                                                                                 | `Timestamp`                |
+| `java.util.Date`, `time`, `java.sql.Types.TIME`, `java.sql.Time`                                                                                                                                                                                                                                                                                                                                                                                                                                              | `Datetime`                 |
```
[the remark this answers is not in the document]
Comment (KirillKurdyukov) [reply ydb/docs/en/core/integrations/liquibase.md:41]: TzDate, TzDatetime, TzTimestamp not supported in table columns

[the remark this answers is not in the document]
Comment (KirillKurdyukov) [reply ydb/docs/en/core/integrations/liquibase.md:41]: I take type mapping from JDBC.

https://github.com/ydb-platform/ydb-jdbc-driver/blob/develop/jdbc/src/main/java/tech/ydb/jdbc/impl/YdbTypesImpl.java#L94

In YDB Java SDK Timestamp corresponds to the Instant.class [a.k.a java.sql.Timestamp].

File: ydb/docs/en/core/integrations/liquibase.md [hunk 26]
```diff
@@ -0,0 +1,551 @@
...
+| `Interval`                              | [Format](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/Duration.html#parse(java.lang.CharSequence)) `java.time.Duration`, prefix `PT` is optional                                                                                                                                                                                                                                                                                       |
+| `Date`, `Datetime`, `Timestamp`         | Format [ISO-8601](https://ru.wikipedia.org/wiki/ISO_8601), in Java, the following format is used: [ISO_LOCAL_DATE](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE), [ISO_LOCAL_DATE_TIME](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME), [ISO_INSTANT](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_INSTANT) |
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:298]: it's still not clear which of these formats you're supposed to put into a CSV file to be imported + in which precision (YDB's or Java's) as they are different + what happens with timezones

Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:298]:
```suggestion
| `Date`, `Datetime`, `Timestamp`         | Format [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601), in Java, the following format is used: [ISO_LOCAL_DATE](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE), [ISO_LOCAL_DATE_TIME](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME), [ISO_INSTANT](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_INSTANT) |
```

File: ydb/docs/en/core/integrations/liquibase.md [hunk 27]
```diff
@@ -0,0 +1,551 @@
...
+| `Float`, `Double`, `Decimal(22, 9)`     | A real number                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `Interval`                              | [Format](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/Duration.html#parse(java.lang.CharSequence)) `java.time.Duration`, prefix `PT` is optional                                                                                                                                                                                                                                                                                       |
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:297]: This `PT` thing is part of ISO8601 too, and there might be days/weeks/years between P and T, and intervals can be negative, so it's not just a prefix. So if it follows the standard it's better to refer to it instead of Java docs. Please clarify what exactly is optional. Are there any additional limitations like in YDB itself?

File: ydb/docs/en/core/integrations/liquibase.md [hunk 28]
```diff
@@ -0,0 +1,551 @@
...
+
+{% list tabs %}
+
+- Programmatically from Java / Kotlin applications
+
+  The project's [README](https://github.com/ydb-platform/ydb-java-dialects/tree/main/liquibase-dialect) describes how to use it from Java or Kotlin in detail. There is also an [example of a Spring Boot application](https://github.com/ydb-platform/ydb-java-examples/tree/master/jdbc/spring-liquibase-app) using it.
+
+- Liquibase CLI
+
+  First, you need to install Liquibase itself using [one of the recommended methods](https://docs.liquibase.com/start/install/home.html). Then you need to place the `.jar` archives of [{{ ydb-short-name }} JDBC driver](https://github.com/ydb-platform/ydb-jdbc-driver/releases) and Liquibase [{{ ydb-short-name }} dialect](https://mvnrepository.com/artifact/tech.ydb.dialects/liquibase-ydb-dialect/1.0.0) into the `internal/lib` folder.
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:324]: btw isn't the official maven repo this one https://repo.maven.apache.org/maven2/tech/ydb/ ? and the referenced one seems to be some unofficial one

File: ydb/docs/en/core/integrations/liquibase.md [hunk 29]
```diff
@@ -0,0 +1,551 @@
...
+| kurdyukov-kir | ""              |          | 12:53:28     | 1544007500    | createIndex indexName=index_episodes_title, tableName=episodes | EXECUTED | migration/episodes.xml | index_episodes_title |        | 4.25.1    | 9:49b8b0b22d18c7fd90a3d6b2c561455d | 2             |
+
+### Database schema evolution
+
+Let's say we need to create a {{ydb-short-name }} topic and turn off the param `AUTO_PARTITIONING_BY_SIZE` of the table. This can be done with a native SQL script:
+
+```sql
+--liquibase formatted sql
+
+--changeset kurdyukov-kir:create-a-topic
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:421]: why your login is used here?

File: ydb/docs/en/core/integrations/liquibase.md [hunk 30]
```diff
@@ -0,0 +1,551 @@
...
+
+```bash
+liquibase changelog-sync --changelog-file=changelog.xml 
+```
+
+The result will be liquibase synchronization in your project:
+
+![_assets/liquibase-step-4.png](_assets/liquibase-step-4.png)
+
+## Connecting to YDB
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:538]:
```suggestion
## Connecting to {{ ydb-short-name }}
```

File: ydb/docs/en/core/integrations/liquibase.md [hunk 31]
```diff
@@ -0,0 +1,551 @@
...
+  First, you need to install Liquibase itself using [one of the recommended methods](https://docs.liquibase.com/start/install/home.html). Then you need to place the `.jar` archives of [{{ ydb-short-name }} JDBC driver](https://github.com/ydb-platform/ydb-jdbc-driver/releases) and Liquibase [{{ ydb-short-name }} dialect](https://mvnrepository.com/artifact/tech.ydb.dialects/liquibase-ydb-dialect/1.0.0) into the `internal/lib` folder.
+
+  ```bash
+  # $(which liquibase)
+  cd ./internal/lib/
+  
+  # you may need to sudo
+  # set an actual versions .jar files
+  curl -L -o ydb-jdbc-driver.jar https://repo1.maven.org/maven2/tech/ydb/jdbc/ydb-jdbc-driver-shaded/2.0.7/ydb-jdbc-driver-shaded-2.0.7.jar
+  curl -L -o liquibase-ydb-dialect.jar https://repo1.maven.org/maven2/tech/ydb/dialects/liquibase-ydb-dialect/1.0.0/liquibase-ydb-dialect-1.0.0.jar
```
Comment (blinkov) [inline ydb/docs/en/core/integrations/liquibase.md:333]: It seems like maven.org also belongs to some commercial company instead of ASF

**Review by blinkov — DISMISSED** [review]

File: ydb/docs/en/core/integrations/liquibase.md (hunk 29 above)
Comment (KirillKurdyukov) [reply ydb/docs/en/core/integrations/liquibase.md:421]: ${author}:${id}

File: ydb/docs/en/core/integrations/liquibase.md (hunk 31 above)
Comment (KirillKurdyukov) [reply ydb/docs/en/core/integrations/liquibase.md:333]: Changed on `https://repo.maven.apache.org/maven2/tech/`

**Review by fomichev3000 — APPROVED** [review]
